### PR TITLE
[dv] Clean up for phase_read_to_end

### PR DESCRIPTION
**Pull request by weicaiyang**
1. Remove raise/drop objection in scb/mon
2. Update mon to use monitor_ready_to_end
Signed-off-by: Weicai Yang <weicai@google.com>

### CODE REVIEW

**Comment by weicaiyang** [comment]
@cindychip 
I didn't update the alert_esc_agent. Can you update it? Thanks

File: hw/ip/uart/dv/env/uart_env_cfg.sv [hunk 1]
```diff
@@ -29,6 +29,9 @@ class uart_env_cfg extends cip_base_env_cfg #(.RAL_T(uart_reg_block));
     num_interrupts = ral.intr_state.get_n_used_bits();
     // only support 1 outstanding TL item
     m_tl_agent_cfg.max_outstanding_req = 1;
+    // there is a forever loop to read interrupts for check in seq. No need to extend sim time
```
Comment (sriyerg) [inline hw/ip/uart/dv/env/uart_env_cfg.sv:32]: Is this related to `do_clear_all_interrupts` knob? What if someone sets this to 0 as a plusarg?

File: hw/dv/sv/scoreboard/scoreboard.sv [hunk 2]
```diff
@@ -208,9 +207,6 @@ class scoreboard#(type ITEM_T = uvm_object,
             `uvm_info(get_full_name(), $sformatf(
                       "Scoreboard timeout caused by packet drop, act/exp items = %0d/%0d",
                       num_of_act_item, num_of_exp_item), UVM_LOW)
-            if (objection_raised) begin
-              run_phase_h.drop_objection(this, $sformatf("%s objection dropped", `gfn));
-              objection_raised = 1'b0;
             end
```
Comment (sriyerg) [inline hw/dv/sv/scoreboard/scoreboard.sv:210]: Extra `end` causing private CI to fail, FYI.

**Comment by cindychip** [comment]
I will update alert_handler. Thanks Weicai

File: hw/ip/uart/dv/env/uart_env_cfg.sv (hunk 1 above)
Comment (weicaiyang) [reply hw/ip/uart/dv/env/uart_env_cfg.sv]: this is for an interrupt polling thread that I created in uart seq. But I actually found that part was changed and the interrupt thread was ended before exiting the seq, so that seq can be added to stress_all. I removed this part as it's not needed.

File: hw/dv/sv/scoreboard/scoreboard.sv (hunk 2 above)
Comment (weicaiyang) [reply hw/dv/sv/scoreboard/scoreboard.sv]: Fixed. Private CI is passing now. thanks.
